README.md

```diff
@@ -370,6 +370,24 @@ change this line in `mix.exs`:
 defp elixirc_paths(:test), do: ["lib", "web", "test/support", "test/factories"]
 ```
 
+## Multi-tenant support
+
+If you are depending on query prefixes to switch between tenant schemas you can 
+pass `repo_options` to your factory which can be used to set the `:prefix` on your insert queries.
+This allows you to create a factory per tenant.
+
+```elixir
+defmodule MyApp.TenantFactory
+  use ExMachina.Ecto, repo: MyApp.Repo, repo_options: [prefix: "test_tenant"]
+
+  def user_factory do
+    ...
+  end
+
+  ...
+end
+```
```

Comment:
🎉

Reply:
Found this while looking for tenet-support fixtures for my own project. If this PR still needs more test, and the above described acceptance criteria are still valid, let me know and maybe I can take a shot. Love using this on my other projects.